internal/app/api/v1/executions.go

```diff
@@ -193,13 +193,13 @@ func (s TestKubeAPI) ExecutionLogsHandler() fiber.Handler {
 func (s TestKubeAPI) GetExecutionHandler() fiber.Handler {
 	return func(c *fiber.Ctx) error {
 		ctx := c.Context()
-		scriptID := c.Params("id", "-")
+		scriptID := c.Params("id", "")
 		executionID := c.Params("executionID")
 
 		var execution testkube.Execution
 		var err error
 
-		if scriptID == "-" {
+		if scriptID == "" {
```

Comment:
remove this branch (handle only execution ID)

Reply:
✅